rviz_common/src/rviz_common/temp/stl_loader.cpp

```diff
@@ -80,13 +80,13 @@ bool STLLoader::load(const std::string & path)
 
   // find the file size
   fseek(input, 0, SEEK_END);
-  long fileSize = ftell(input);  // NOLINT maybe use int64 instead
+  long fileSize = ftell(input); // NOLINT ftell returns long
```

Comment:
Please do not remove the extra space (!) 😢, that is the cpplint style and the preferred way according to the ROS 2 style. Here and elsewhere in the pr.